spotbugs/src/test/java/edu/umd/cs/findbugs/BugRankerTest.java

```diff
@@ -0,0 +1,19 @@
+package edu.umd.cs.findbugs;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class BugRankerTest {
+    /**
+     * @see <a href="https://github.com/spotbugs/spotbugs/issues/1161"></a>
```

Comment:
add a text to link, or hyperlink won't work.

Reply:
Thank you for your review. Added a text https://github.com/spotbugs/spotbugs/pull/1436/commits/913f233b9bd1f8ec22fedd8d8cafd1ee489d9c35